src/main/kotlin/graphics/scenery/volumes/Colormap.kt

```diff
@@ -155,7 +155,7 @@ class Colormap(val buffer: ByteBuffer, val width: Int, val height: Int) {
          */
         @JvmStatic fun list(): List<String> {
             // FIXME: Hardcoded for the moment, not nice.
-            val list = arrayListOf("grays", "hot", "jet", "plasma", "viridis")
+            val list = arrayListOf("grays", "hot", "jet", "plasma", "viridis", "red-blue", "rb-darker")
```

Comment:
Please see that the files you've added don't violate any licenses. Also, the PNG files contain some text and a border - both ought to be removed.

Reply:
I got the colormaps from https://www.ncl.ucar.edu/Document/Graphics/color_table_gallery.shtml. Its an open-source scivis package and the license information is here https://www.ncl.ucar.edu/Download/NCL_source_license.shtml. My interpretation would be that we are allowed to use the files but need to include their license message. This is what they say about the code, not sure it also applies to the png files.

Do you suggest removing the files?

Reply:
This is basically the BSD license, so it's fine, and we'll acknowledge them appropriately 👍

Reply:
I have added an ACKNOWLEDGEMENTS.md file, and also removed the margin text on the colormaps.